pkg/controllers/resources/csidrivers/syncer.go

```diff
@@ -16,8 +16,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func New(ctx *synccontext.RegisterContext) (syncertypes.Object, error) {
-	mapper, err := ctx.Mappings.ByGVK(mappings.CSIDrivers())
+func New(_ *synccontext.RegisterContext) (syncertypes.Object, error) {
```

Comment:
can't we just remove the parameter?

Reply:
this needs to adhere to the create syncer type function that uses this parameter for other syncers